app/views/courses/_basics.html.erb

```diff
@@ -70,6 +70,9 @@
     </div>
   <% else %>
     <%= t('basics.editors') %>
+    <%= helpdesk(t('admin.course.info.no_right_to_change_editors',
+                   project_email: mail_to(DefaultSetting::PROJECT_EMAIL, nil)),
```

Comment:
`name=nil` is the [default](https://apidock.com/rails/ActionView/Helpers/UrlHelper/mail_to), so no need to specify it here explicitly.